### PR TITLE
Save team settings.controls to chart store

### PR DESCRIPTION
**Pull request by elanals**
So that it can be accessed directly in controls. For motivation and more details, see [this ticket](https://www.notion.so/datawrapper/Team-Feature-Customize-selection-in-Custom-format-dropdown-e36719b0bb17410481a06ecd43b94a61).

### CODE REVIEW

File: controller/chart/visualize.php [hunk 1]
```diff
@@ -64,6 +64,8 @@
 
         $res = Hooks::execute('enable_web_to_print');
         $webToPrint = !empty($res) && $res[0] === true;
+        $org = $chart->getOrganization();
+        if ($org) $teamSettingsControls = $org->getSettings("controls") ?? [];
```
Comment (gka) [inline controller/chart/visualize.php:68]:
```suggestion
        if ($org) $teamSettingsControls = $org->getSettings("controls") ?? new stdClass();
```

**Review by gka — COMMENTED** [review]
I think I would just name it `teamSettings`. As far as the chart editor is concerned, that's the only available team setting. If we want to keep it open for more team settings in the future, we can also pass the "full" object structure, but leave only the `controls` key. This way the team settings "paths" would be the same.

```js
teamSettings: { controls: teamSettingsControls }
```

but apart from that it's fine.

**Review by davidkokkelink — APPROVED** [review]

**Comment by elanals** [comment]
@gka, I made changes, can you confirm that this is what you going for?

**Review by gka — APPROVED** [review]
looks good

**Pull request merged**